source/extensions/http/header_validators/envoy_default/path_normalizer.cc

```diff
@@ -172,17 +172,18 @@ PathNormalizer::normalizePathUri(RequestHeaderMap& header_map) const {
   // asterisk-form  = "*"
   //
   // TODO(#23887) - potentially separate path normalization into multiple independent operations.
-  const bool is_connect_method =
-      header_map.method() == ::Envoy::Http::Headers::get().MethodValues.Connect;
-  const bool is_options_method =
+  const bool is_connect_request =
```

Comment:
is_standard_connect_request with comment that we handle extended connect differently?

Reply:
I just moved the check for standard connect down where it is used, as there is a comment there already.